.github/workflows/ci.yml

```diff
@@ -0,0 +1,101 @@
+name: Django Rest Framework Reactive CI
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+*"
+
+  pull_request:
+  schedule:
+    - cron: "30 2 * * *"
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        tox-env: [py36, py37, py38, py39, docs, linters, packaging, migrations]
+        include:
+          - tox-env: py36
+            python-version: 3.6
+          - tox-env: py37
+            python-version: 3.7
+          - tox-env: py38
+            python-version: 3.8
+          - tox-env: py39
+            python-version: 3.9
+          - tox-env: docs
+            python-version: 3.6
+          - tox-env: linters
+            python-version: 3.6
+          - tox-env: packaging
+            python-version: 3.6
+          - tox-env: migrations
+            python-version: 3.6
+
+    services:
+      postgres: 
+        image: postgres:10-alpine
+        env:
+          POSTGRES_USER: drfr
+          POSTGRES_DB: drfr
+          POSTGRES_PASSWORD: drfr
+        ports: 
+          - 5432/tcp
+      
+      redis:
+        image: redis:5-alpine
+        ports: 
+          - 6379/tcp
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip tox
+      - name: Run Tests
+        run: >
+          tox -e ${{ matrix.tox-env }}
+          --skip-missing-interpreters false
+          --recreate
+        env:
+          DRFR_REDIS_PORT: ${{ job.services.redis.ports[6379] }}
+          DRFR_POSTGRESQL_HOST: localhost
+          DRFR_POSTGRESQL_NAME: drfr
+          DRFR_POSTGRESQL_USER: drfr
+          DFRF_POSTGRESQL_PASS: drfr
+          DRFR_POSTGRESQL_PORT: ${{ job.services.postgres.ports[5432] }}
+      - name: Upload coverage to Codecov
+        if: matrix.tox-env == 'py36' || matrix.tox-env == 'py37' || matrix.tox-env == 'py38' || matrix.tox-env == 'py39'
+        uses: codecov/codecov-action@v1
+        with:
+          file: .coverage
+    
+  build:
+    runs-on: ubuntu-20.04
+
+    needs: test
+    
+    if: "startsWith(github.ref, 'refs/tags')"
```

Comment:
Add also `if: github.ref == 'refs/heads/master'` conditional, so tagged pull requests dont trigger this workflow. The workflow would fail due to token not being available in PR workflow.